chain/penumbra/penumbra_chain_test.go

```diff
@@ -15,19 +16,21 @@ func TestPenumbraChainStart(t *testing.T) {
 	}
 
 	t.Parallel()
-	ctx, home, pool, network, err := ibctest.SetupTestRun(t)
-	require.NoErrorf(t, err, "failed to set up test run")
+
+	pool, network := ibctest.DockerSetup(t)
```

Comment:
maybe we keep home as part of the `DockerSetup` just so it will always be the last cleanup item to run?